src/App.jsx

```diff
@@ -9,40 +8,100 @@ function App() {
   const [todoList, setTodoList] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
 
-  useEffect(() => {
-    const promise = new Promise((resolve, reject) => {
-      setTimeout(() => {
-        resolve({
-          data: {
-            todoList: JSON.parse(localStorage.getItem('savedTodoList')) || []
-          }
-        });
-      }, 2000);
-    });
-
-    promise.then((result) => {
-      setTodoList(result.data.todoList);
+  const fetchData = async () => {
+
+    const options = {
+      method: "GET",      
+      headers: {
+        'Authorization': `Bearer ${import.meta.env.VITE_AIRTABLE_API_TOKEN}`
```

Comment:
You could save your self a little space by assigning these to consts at the top of the file. No need to make any change but always be on the lookout for any code that is awkward to read.